### PR TITLE
ruskakimov/mobile_full_screen_dialog

### PR DESCRIPTION
**Pull request by ruskakimov**
#### [Demo](https://ruskakimov.github.io/binary-static/br_mobile_full_screen_dialog/en/trade.html)
(click on **show modal** button)
## Props:
- `title` - dialog title
- `visible` - bool that controls dialog visibility
- `onClose` - callback called on dialog closing (triggered after animation ends)

## Usage example:
```jsx
<FullscreenDialog
    title='Select asset'
    visible={this.state.modalIsVisible}
    onClose={() => this.setState({
        modalIsVisible: false
    })}
>
    <p>some content</p>
</FullscreenDialog>
```
### Behaviour:
- header is fixed
- body is not scrollable when dialog is open
- scroll on overflow
- component has no state, renders based on `visible` prop

### CODE REVIEW

File: src/sass/app_2.scss [hunk 1]
```diff
@@ -8,4 +8,6 @@
 @import 'app_2/_common/buttons.scss';
 @import 'app_2/_common/input_field.scss';
 @import 'app_2/_common/dropdown.scss';
+@import 'app_2/_common/contracts_popup.scss';
```
Comment (zakibinary) [inline src/sass/app_2.scss]: Do we need to call `contracts_popup.scss` for this component?

Comment (ruskakimov) [reply src/sass/app_2.scss]: nope it doesn't depend on it. I thought it got here from the merge conflict, but I don't see any merges in commit history. That's weird. 😅

Comment (zakibinary) [reply src/sass/app_2.scss]: I guess it should be removed then. :)

**Comment by ruskakimov** [comment]
now part of react_trading_mobile_widget branch

**Pull request closed without merging**